elastic/beats
# Cherry-pick #15097 to 7.x: [Metricbeat] [AWS] Add DynamoDB AWS light module

Cherry-pick of PR #15097 to 7.x branch. Original message: 

This PR implements `DynamoDB` AWS metricbeat Module of https://github.com/elastic/beats/issues/14856

Reference: https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/metrics-dimensions.html

Dashboard:
![Screenshot 2020-01-08 at 12 10 39](https://user-images.githubusercontent.com/11754898/71970154-c7bae000-3210-11ea-9d73-d851fbb1b6c6.png)

### Manual Testing

1. Enable aws module
2. Use the following configuration:
```
- module: aws
  period: 60s
  metricsets:
    - dynamodb
  credential_profile_name: metricbeat
  regions:
    - eu-central-1
```
3. Start Metricbeat
4. Check that events are sent to ES or any other output.
Notes:
1. This requires an AWS account with DynamoDB features
2. In order to automatically populate realistic data in AWS, DM author for instructions.
